frontend/src/global_styles/content/_toast.sass

```diff
@@ -36,23 +36,24 @@ $nm-border-radius: rem-calc(2px)
 $nm-box-padding: rem-calc(10px 35px 10px 35px)
 $nm-toaster-width: rem-calc(550)
 
-$nm-color-error-background: #fedada
 
-$nm-color-success-border: #35c53f
-$nm-color-success-icon: #35c53f
-$nm-color-success-background: #d8fdd1
+$nm-color-success-border: var(--borderColor-success-muted)
+$nm-color-success-icon: var(--fgColor-success)
+$nm-color-success-background: var(--bgColor-success-muted)
 
-$nm-color-warning-border: #ef9e56
-$nm-color-warning-icon: #ef9e56
-$nm-color-warning-background: #ffe6c6
+$nm-color-warning-border: var(--borderColor-attention-muted)
+$nm-color-warning-icon: var(--fgColor-attention)
+$nm-color-warning-background: var(--bgColor-attention-muted)
 
-$nm-color-info-icon: #155282
-$nm-color-error-icon: var(--content-form-danger-zone-bg-color)
+$nm-color-error-icon: var(--fgColor-danger)
+$nm-color-error-background: var(--bgColor-danger-muted)
+$nm-color-error-border: var(--borderColor-danger-muted)
 
-$nm-color-border: #8fc0db
-$nm-color-background: #e3f5ff
-$nm-padding-box: rem-calc(10px)
+$nm-color-info-icon: var(--fgColor-accent)
+$nm-color-border: var(--borderColor-accent-muted)
+$nm-color-background: var(--bgColor-accent-muted)
```

Comment:
The muted styles are semi transparent which looks weird on the notifications
<img width="664" alt="Bildschirmfoto 2024-06-12 um 07 22 56" src="https://github.com/opf/openproject/assets/7457313/e2bdb7eb-4269-489d-925e-8fed7fd56f53">

Reply:
yeah I agree. I checked the Primer Design System, they are using muted variables for banners. I used the same variables as they used, and it works well on high-contrast and light modes, but for dark mode it's a bit weird. Do we want to use primer variables for toasts?

Reply:
What does it mean "it's a bit weird"?

Reply:
I mean, with a transparent background, elements behind the notification are visible, so it is weird for users.

So for now, I will revert the changes for colors and will talk with products to see which primer color variable is suitable for toasts.